main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-steplib/steps-expo-detach/expo"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+// Config ...
+type Config struct {
+	ProjectPath    string          `env:"project_path,dir"`
+	ExpoCLIVersion string          `env:"expo_cli_verson,required"`
+	UserName       string          `env:"user_name"`
+	Password       stepconf.Secret `env:"password"`
+	Logout         bool            `env:"logout,opt[true,false]"`
```

Comment:
I think logout is necessary. It should not be optional. Also I would do it in defer just in case we log the user out properly.